src/datadir-tests

```diff
@@ -32,15 +50,14 @@ foreach ($finder as $testSuite) {
     $temp = new Temp($testSuite->getBasename());
     $temp->initRunFolder();
 
-    $fs = new Filesystem();
     $fs->mirror($testSuite->getPathname() . '/source/data', $temp->getTmpFolder());
 
     $fs->mkdir($temp->getTmpFolder() . '/in/tables', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/in/files', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/out/tables', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/out/files', 0777);
 
-    $runCommand = "KBC_DATADIR={$temp->getTmpFolder()} php /code/src/run.php";
+    $runCommand = "KBC_DATADIR={$temp->getTmpFolder()} php " . $script;
```

Comment:
chybí mi example v usage jak mám použít ten custom run script.

Reply:
```
λ php src\datadir-tests -h
Usage:
        datadir-tests /path/to/tests script-to-run.php

All parameters are optional. Default values are "tests/functional" and "src/run.php"
```

Asi nevím, co myslíš. Jakože není jasné, k čemu ten parametr je? 

```
λ php src\datadir-tests -h
Usage:
        datadir-tests [path to tests directory] [script to test]

Example:
        datadir-tests /path/to/tests script-to-run.php

All parameters are optional. Default values are "tests/functional" and "src/run.php"
```

Takhle lepší?

Reply:
Jee, tohodle helpu jsem si nevšiml, hledal jsem to v README :-)

Reply:
Upraveno na obou místech.